task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +115,16 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+        for (let i = 0; i < attempts;) {
```

Comment:
looks like while-loop with extra steps. You added **i** variable but it's always 0, what's the point of using it?

Try to rewrite it like **for (let i = 0; i < attempts; i++)** and without modifiying **attempts**

Reply:
I think something like this